src/js/tabs/trade.controller.js

```diff
@@ -615,8 +615,13 @@ TradeTab.prototype.angular = function(module)
       tx.on('success', function(res) {
         setEngineStatus(res, true, type);
 
+
         if (!modifying) order.mode = 'done';
 
+        $timeout(function() {
```

Comment:
No, the notification goes away after 2 seconds.  The form is not displayed until 2 seconds.